pkg/azuredisk/azure_common_test.go

```diff
@@ -180,6 +214,35 @@ func TestNormalizeStorageAccountType(t *testing.T) {
 	}
 }
 
+func TestNormalizeCachingMode(t *testing.T) {
+	tests := []struct {
+		desc        string
+		req         v1.AzureDataDiskCachingMode
+		expectedErr error
+	}{
+		{
+			desc:        "CachingMode not exist",
+			req:         "",
+			expectedErr: nil,
+		},
+		{
+			desc:        "Not supported CachingMode",
+			req:         "WriteOnly",
+			expectedErr: fmt.Errorf("azureDisk - WriteOnly is not supported cachingmode. Supported values are [None ReadOnly ReadWrite]"),
+		},
+		{
+			desc:        "Valid CachingMode",
+			req:         "ReadOnly",
+			expectedErr: nil,
+		},
+	}
+	for _, test := range tests {
+		_, err := normalizeCachingMode(test.req)
```

Comment:
pls also check return value, not only error

Reply:
ok